api/types/database.go

```diff
@@ -479,6 +482,21 @@ func (d *DatabaseV3) IsCloudHosted() bool {
 	return d.IsAWSHosted() || d.IsCloudSQL() || d.IsAzure()
 }
 
+// GetCloud gets the cloud this database is running on, or an empty string if it
+// isn't running on a cloud provider.
+func (d *DatabaseV3) GetCloud() string {
```

Comment:
nit: Are we going to support other types of cloud-hosted databases? Such as Snowflake and MongoDB atlas, which are hosted in different "cloud providers".

Reply:
I don't think we need to until we have auto-discovery for Snowflake and MongoDB atlas databases.